File: DSA-JAVA/Linear_Search.java

public class Linear_Search {

    public static void main(String[] args) {
        int[] array = {1, 3, 5, 8, 2, 4, 9};
        int target = 10;
        boolean found = false;
        for (int i = 0; i < array.length; i++) {
            if (target == array[i]) {
                System.out.println("Element found at index: " + i);
                found = true;
                break;
            }
        }
        if (found != true) {
            System.out.println("Element not in array");
        }

    }
}

File: DSA-C/Linear_Search.c
#include<stdio.h>
#include<stdbool.h>
int main()
{
int target = 2;
bool found = false;
int arr[] = {1, 4, 7, 3, 9, 5, 2, 8};
int len = sizeof(arr) / sizeof(arr[0]);
for(int i = 0; i < len; i++)
{
    if(target == arr[i])
    {
        printf("Target found at index: %d\n", i);
        found = true;
    }
}
if(found == false)
{
printf("Element not in array\n");
}
getchar();
return 0;
}